webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -2063,6 +2129,14 @@
 #
 ##########################################################
 
+# TODO: remove this standoff class one it is not referenced anymore (biblio mapping)
```

Comment:
@SepidehAlassi I originally created this for the wiki html tag which I got rid off. 

What would be an appropriate description (comment and label) for your use in biblio?